### ISSUE TITLE
Adaptive dropout for training deep neural networks

### ISSUE THREAD

**Issue by shunk031** [shunk031/paper-survey#110]
https://papers.nips.cc/paper/5032-adaptive-dropout-for-training-deep-neural-networks.pdf

**Comment by shunk031** [comment]
https://shunk031.github.io/paper-survey/paper-summary/CV/Adaptive_Dropout_for_Training_Deep_Neural_Networks
